content/docs/command-reference/pull.md

```diff
@@ -167,16 +164,14 @@ $ dvc pull train.dvc
 > Please delete the `.dvc/cache` directory first (with `rm -Rf .dvc/cache`) to
 > follow this example if you tried the previous ones.
 
-Our [pipeline](/doc/command-reference/pipeline) has been setup with these
+Our [pipeline](/doc/command-reference/dag) has been setup with these
 [stages](/doc/command-reference/run):
 
 ```dvc
-$ dvc pipeline show evaluate.dvc
-data/data.xml.dvc
-prepare.dvc
-featurize.dvc
-train.dvc
-evaluate.dvc
+$ dvc dag src/evaluate.py.dvc
++---------------------+
+| src/evaluate.py.dvc |
++---------------------+
 ```
```

Comment:
Changing the output here because it's a relatively less incorrect output than the previous one. 

Unfortunately, the remote still isn't working. I cannot find the output of `evaluate.dvc` since it doesn't exist. Also, files in `src/` are Git-tracked files, so I had to remove them from git. Even after adding all files (`dvc add`) in `src/`, `dvc dag evaluate.py.dvc` does not show the linked files either. I suppose I am missing something and having the remote changed would result in an accurate output. This output is the output I'm currently getting.

Reply:
Yeah that's totally wrong 😋 please read the cmd ref at https://dvc.org/doc/command-reference/dag and notice the https://github.com/iterative/example-get-started project has been updated for 1.x already. There are no .dvc stage files, it's all in dvc.yaml. Have you followed the new Get Started at https://dvc.org/doc/start?

Reply:
Also you don't need to use a target in `dvc dag`, just the plan command should be enough... But it will print a diagram which may be too long. If it seems to (too long) just use `cat dvc.yaml` here as well. Thanks

Reply:
Never mind. cat dvc.yaml will bee way too long here also. Let's try `dvc dag` with the right target please. Lmk if you need help... But do read those refs I shared, hopefully you'll get it from that. Thanks

Reply:
Alright, I've updated it with `dvc dag evaluate`, but it still gave the entire diagram as output.
Because the `deps` files link all the way back to `prepare` and `featurize`.

Reply:
OK. Yeah I contradicted myself... You don't need to use a target in `dvc dag`. The plan command should show everything by default. I think we're gonna have to go with a simple list of stage names like in push, because this diagram is too huge.

But the rest of the example will also need updating here, so it makes sense. Similar to the changes in https://github.com/iterative/dvc.org/pull/1591/files. Please try to make sure the entire context makes sense and works. Thanks

Reply:
> The rest of the example will also need updating here, so it makes sense (similar to the changes in https://github.com/iterative/dvc.org/pull/1591/files)

Didn't get you on this one, actually. :thinking: The changes in #1591 are only limited to `push.md`, and `pull.md` uses the Get-Started repo as the example. So why can't I just list the stage names like this:
```
prepare
featurize
train
evaluate
```
? :thinking:

Reply:
Oh okay, I got your point. But a problem arises as stated in [#1584(comment)](https://github.com/iterative/dvc.org/pull/1584#issuecomment-663359221).

Reply:
I don't see the issue. It's just a matter of updating the rest of the example please.

Reply:
This is the other missing point but we can extract into a separate issue. Are you having trouble understanding what I meant here about updating the rest of the example @utkarshsingh99 ? Thanks